governance/votes/Executive vote - April 26, 2021.md

```diff
@@ -0,0 +1,126 @@
+	* Introduces a second Collateral Greenlight poll date on the 1st Monday of each month.
+* [MIP4c2-SP13: MIP0 Amendments](https://forum.makerdao.com/t/mip4c2-sp13-mip0-amendments/6758)
+	* Adds a procedure for tagging MIP Sets and clarifies the responsibilities of relevant actors.
+
+### Declarations of Intent
+
+* [MIP13c3-SP10: Declaration of Intent - eurDai](https://forum.makerdao.com/t/mip13c3-sp10-declaration-of-intent-eurdai/6766)
+	*  Intention to investigate and build a subprotocol providing a DAI-like stablecoin pegged to the official currency of the European Union.
+
+### Core Unit MIP Sets
+
+* MKT-001, Content Production Core Unit Proposals 
+	* [MIP39c2-SP5: Content Production Core Unit, MKT-001](https://forum.makerdao.com/t/mip39c2-sp5-content-production-core-unit-mkt-001/6823)
+	* [MIP40c3-SP5: Core Unit Budget - MKT-001](https://forum.makerdao.com/t/mip40c3-sp5-core-unit-budget-mkt-001/6824)
+	* [MIP41c4-SP5: Facilitator Onboarding, MKT-001](https://forum.makerdao.com/t/mip41c4-sp5-facilitator-onboarding-mkt-001/6825)
+* GRO-001, Growth Core Unit Proposals 
+	* [MIP39c2-SP4: Growth Core Unit, GRO-001](https://forum.makerdao.com/t/mip39c2-sp4-growth-core-unit-gro-001/6715)
+	* [MIP40c3-SP4: Core Unit Budget, GRO-001](https://forum.makerdao.com/t/mip40c3-sp4-core-unit-budget-gro-001/6716)
+	* [MIP41c4-SP4: Facilitator Onboarding, GRO-001](https://forum.makerdao.com/t/mip41c4-sp4-facilitator-onboarding-gro-001/6717)
+* MDS-001, MakerDAO Shop Core Unit Proposals 
+	* [MIP39c2-SP6: MakerDAO Shop Core Unit, MDS-001](https://forum.makerdao.com/t/mip39c2-sp6-makerdao-shop-core-unit-mds-001/6827)
+	* [MIP40c3-SP6: MakerDAO Shop Budget, MDS-001](https://forum.makerdao.com/t/mip40c3-sp6-makerdao-shop-budget-mds-001/6829)
+	* [MIP41c4-SP6: MakerDAO Shop Facilitator Onboarding, MDS-001](https://forum.makerdao.com/t/mip41c4-sp6-makerdao-shop-facilitator-onboarding-mds-001/6828)
+* PE-001, Protocol Engineering Core Unit Proposals 
+	* [MIP39c2-SP7: Adding Protocol Engineering Core Unit](https://forum.makerdao.com/t/mip39c2-sp7-adding-protocol-engineering-core-unit/6831)
+	* [MIP40c3-SP7: Modify Protocol Engineering Core Unit Budget](https://forum.makerdao.com/t/mip40c3-sp7-modify-protocol-engineering-core-unit-budget/6832)
+	* [MIP41c4-SP7: Facilitator Onboarding, Protocol Engineering Core Unit](https://forum.makerdao.com/t/mip41c4-sp7-facilitator-onboarding-protocol-engineering-core-unit/6833)
+
+**Note: Upon passage of this vote the following distributions would also occur**
+
+* 510,000 DAI to 0xe2c16c308b843eD02B09156388Cb240cEd58C01c, the Protocol Engineering Core Unit Multisig.
+* 1.3M DAI to 0x83e36aAA1c7b99E2D3d07789F7b70FCe46f0d45E, the Protocol Engineering Continuous Operation Multisig, which is only to be used when the protocol cannot provide pre-approved funding. 
+	* **This Multil-Sig requires 3 of 6 signatories and is administered by:**
```

Comment:
Going to assume this is correct. It should be something that Derek signs off on.

Reply:
Also 'Multil' -> 'Multi'

Reply:
Grammarly might be able to pick up that sort of thing.

Reply:
It missed it! Might have been because I entered the markdown text rather than the display text, will keep an eye out for that